### DISCUSSION TITLE
Pb : ESP32 Series SoCs : ESP32-D0WDQ6 & tasmota32-webcam v9.4.0

### DISCUSSION THREAD

**Discussion by niconow** [arendst/Tasmota#11934] [Q&A] [answered]
Hello,

I want to share my experience and eventually had advice with ESP32-CAM  module (with an ESP32 Series SoCs : ESP32-D0WDQ6). I use ESP-Flasher on Mac.
When i load tasmota32.bin (v9.4.0). I have access on web page and i can configure like AITHINKER CAM.
-> result : On web page I don't have the video but i can control correctly the LED.
 
When i load tasmota32-webcam.bin (v9.4.0), the program never start it reset in loop (cf. logs attach). 
(I have check the power supply)

If someone have any idea or had already test this version.
I attach logs.
[LogErrortasmota32webcam_reset.pdf](https://github.com/arendst/Tasmota/files/6394620/LogErrortasmota32webcam_reset.pdf)
![Capture d’écran 2021-04-28 à 22 08 28](https://user-images.githubusercontent.com/18490116/116468722-81f4b180-a871-11eb-936d-9143d22f9c4c.png)

Thanks for your help.

Ps : Tasmota is really great, congratulation. You can be proud 💯

**Comment by barbudor** [comment, marked as answer]
The ESP32 model is that same as on the AI-Thinker
However if you can't confirm than yours is an original AI-Thinker ESP32-CAM, it may be different. Tasmota-webcam is assuming the way the web cam is connected to the ESP, if your module is different, it could be the problem.

I would suggest to try it with sample ESP32-CAM Arduino code to check if that works.
